invenio_communities/invitations/services/service.py

```diff
@@ -73,3 +58,16 @@ def create(self, identity, data, uow=None):
         )
 
         return request_item
+
+    def _assert_can_invite(self, identity, entity_dict):
```

Comment:
Will be done in https://github.com/inveniosoftware/invenio-communities/issues/391